### PR TITLE
Adding SECURITY.md for ActiveMQ

### PR DESCRIPTION
**Pull request by coheigea**
It's good security practice to add a SECURITY.md to a github repo, containing details of supported versions, where to find CVEs and how to report new issues.

Feel free to modify as required.

### CODE REVIEW

**Review by jbonofre — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.16.x   | :white_check_mark: |
+| 5.15.x   | :white_check_mark: |
+| < 5.15.x | :x:                |
```
Comment (gemmellr) [inline SECURITY.md:9]: This seems to be in disagreement with what has been discussed and agreed long-ago, and so feels potentially misleading at this point. Certainly worthy of discussion before merging a 'policy' PR after 4 minutes at a time most folks arent around and likely just before announcing a new release that seems to disagree with it.

5.15.15 was discussed, agreed, and finally announced to be the last 5.15.x release over 9 months ago. 5.16.3 occurred 3 months later and contained various dep updates with security fixes, 5.16.4 is about to occur over 9 months later doing the same again, with no equivalent 5.15.x update release being made or ever mentioned in a discussion.

Comment (jbonofre) [reply SECURITY.md:9]: @gemmellr I will change 5.15.x. I just missed it. Nothing changed to what have been discussed on the mailing list already.

Comment (gemmellr) [reply SECURITY.md:9]: Ok great. I think it is also past time it was removed from the website too, so perhaps while adding 5.16.4?

Comment (jbonofre) [reply SECURITY.md:9]: Good idea, I will do that on website as part of the 5.16.4 update.

**Pull request merged**